src/main/java/org/prebid/server/proto/openrtb/ext/request/algorix/ExtImpAlgorix.java

```diff
@@ -0,0 +1,20 @@
+package org.prebid.server.proto.openrtb.ext.request.algorix;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+/**
+ * Algorix Ext Imp
+ * @author xunyunbo@algorix.co
+ */
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ExtImpAlgorix {
+
+    @JsonProperty("sid")
```

Comment:
We use `@JsonProperty(` because as default we use snake case, so here it can be removed